lua/lspconfig/perlls.lua

```diff
@@ -23,12 +23,12 @@ configs.perlls = {
   docs = {
     package_json = "https://raw.githubusercontent.com/richterger/Perl-LanguageServer/master/clients/vscode/perl/package.json";
     description = [[
-    https://github.com/richterger/Perl-LanguageServer/tree/master/clients/vscode/perl
+https://github.com/richterger/Perl-LanguageServer/tree/master/clients/vscode/perl
```

Comment:
These links looks the same to me. What is the change, just the indentation?

Reply:
yeah, I forgot to change the PR title. Initially, the link was in a codeblock in `CONFIG.md` so I couldn't click on it.